uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-work-patterns.html

```diff
@@ -1 +1,56 @@
-Work Patterns
+<div class="btn-group">
+  <button type="button" class="btn btn-info">
+    <span class="glyphicon glyphicon-plus" aria-hidden="true"></span> ADD CUSTOM WORK PATTERN
+  </button>
+</div>
+
+<table class="table table-transparent">
+  <thead>
+  <tr>
+    <th>Default Work Pattern</th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td>Compressed hours 40h - Mon to Thurs</td>
+  </tr>
+  </tbody>
+</table>
+
+<br><br>
```

Comment:
As you can see from the mocks that at a time either Custom or Default work patterns will be shown, so when we make this template dynamic we wont need the `<br>` anymore. This is just temporary to keep some spacing between custom and Default work patterns block.